pom.xml

```
@@ -16,10 +16,18 @@
             <url>https://www.soapui.org/repository/maven2</url>
         </repository>
     </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>smartbear-plugin-repository</id>
+            <url>http://www.soapui.org/repository/maven2</url>
```

<url>http://www.soapui.org/repository/maven2</url>
```suggestion
            <url>https://www.soapui.org/repository/maven2</url>
```

---

this is must have